ruby/tests/common_tests.rb

```diff
@@ -713,6 +715,8 @@ def test_enum_lookup
     assert proto_module::TestEnum::B == 2
     assert proto_module::TestEnum::C == 3
 
+    assert proto_module::TestEnum::V0 == 4
```

Comment:
Can you add matching assertions for `lookup()` and `resolve()` for this field as well, parallel to what appears below for the other elements of this enum?

Reply:
Get it. Added.